From 648402d2c0144a82ca8fd0b81a8ee0d3780b00db Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:36:51 -0400
Subject: [misc] mei: me: downgrade two errors to debug level

Message-id: <1399041470-21330-19-git-send-email-prarit@redhat.com>
Patchwork-id: 79306
O-Subject: [RHEL7.1 PATCH BZ 1088953 18/77] mei: me: downgrade two errors to debug level
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit ed6f7ac1dcae0b95ef2946067beb4ffaba415119
Author: Paul Bolle <pebolle@tiscali.nl>
Date:   Thu Oct 17 13:52:10 2013 +0300

    mei: me: downgrade two errors to debug level

    The mei_me driver prints "suspend" at error level at each suspend. It
    also prints "stop" at error level at driver unload. Downgrade these
    uninteresting messages to debug level.

    Cc: Adam Jackson <ajax@redhat.com>
    Cc: Joe Perches <joe@perches.com>
    Signed-off-by: Paul Bolle <pebolle@tiscali.nl>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/pci-me.c b/drivers/misc/mei/pci-me.c
index ec64eab..862f90b 100644
--- a/drivers/misc/mei/pci-me.c
+++ b/drivers/misc/mei/pci-me.c
@@ -267,7 +267,7 @@ static void mei_me_remove(struct pci_dev *pdev)
  hw = to_me_hw(dev);
 
 
- dev_err(&pdev->dev, "stop\n");
+ dev_dbg(&pdev->dev, "stop\n");
  mei_stop(dev);
 
  mei_pdev = NULL;
@@ -299,7 +299,7 @@ static int mei_me_pci_suspend(struct device *device)
  if (!dev)
   return -ENODEV;
 
- dev_err(&pdev->dev, "suspend\n");
+ dev_dbg(&pdev->dev, "suspend\n");
 
  mei_stop(dev);
 
-- 
1.7.1